pkg/cmd/step/create/test_data/step_create_task/loop-in-buildpack-syntax/pipeline.yml

```diff
@@ -27,7 +28,6 @@ spec:
       inputs:
       - name: workspace
         resource: abayer-jx-demo-qs-master
-    retries: 0
```

Comment:
?

Reply:
No longer needs to be specified in `Pipeline`s. =)

Reply:
ok